.github/workflows/ci.yml

```diff
@@ -201,19 +201,18 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - uses: actions-rs/cargo@v1
         with:
-          command: install
-          args: cargo-nextest
-      - uses: actions-rs/cargo@v1
-        with:
-          command: nextest
-          args: run --features testing --retries 10
```

Comment:
if you change this nearly 80% of our CI will fail, some tests are too sensible for the CI and so fail often. You can see it with the tests on this PR.

Reply:
The tools you are using takes more than the current space we have in the runners

Reply:
Yep, we need more powerful runners, I'll mark this PR as blocked